gulp-jade/gulp-jade.d.ts

```diff
@@ -0,0 +1,87 @@
+// Type definitions for gulp-jade
+// Project: https://github.com/phated/gulp-jade
+// Definitions by: berwyn <https://github.com/berwyn>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "gulp-jade" {
+
+  function GulpJade(params?: GulpJade.Params): any;
+
+  module GulpJade {
+    interface Params {
+        /*******
+        * JADE API OPTIONS
+        *******/
+        
+        /**
+        * If the doctype is not specified as part of the 
+        * template, you can specify it here. It is sometimes
+        * useful to get self-closing tags and remove mirroring
+        * of boolean attributes.
+        */
+        doctype?: string;
+        
+        /**
+        * Adds whitespace to the resulting html to make it
+        * easier for a human to read using '  ' as indentation. 
+        * If a string is specified, that will be used as 
+        * indentation instead (e.g. '\t').
+        */
+        pretty?: any;
```

Comment:
Yeah, I wasn't sure how that was represented in TypeScript declarations. I'll add a commit later to do that, thanks!